skunkworks_crow/src/main/java/org/odk/share/activities/ReceiverActivity.java

```diff
@@ -397,19 +399,25 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                 // request was canceled...
                 Timber.i("QR code scanning cancelled");
             } else {
-                Timber.d("RESULT " + result);
+                Timber.d("Result: %s", result);
                 try {
                     JSONObject obj = new JSONObject(result.getContents());
-                    ssidScanned = (String) obj.get(SSID);
+                    ip = (String) obj.get(IP);
+                    wifiNetworkSSID = (String) obj.get(SSID);
                     port = (Integer) obj.get(PORT);
                     isProtected = (boolean) obj.get(PROTECTED);
                     if (isProtected) {
                         passwordScanned = (String) obj.get(PASSWORD);
                     }
 
-                    Timber.d("Scanned results " + ssidScanned + " " + port + " " + isProtected + " " + passwordScanned);
+                    Timber.d("Scanned results " + wifiNetworkSSID + " " + port + " " + isProtected + " " + passwordScanned);
```

Comment:
I suggest removing all the test log if they are only used for debugging, or just comment them.